File: MODUL 6/CLASS/PRAK605-2410817110003-FerianDestaAshadi.c
#include <stdio.h>

int main() {
    int n, i, j, k;

    printf("");
    scanf("%d", &n);

    int A[n][n], B[n][n], C[n][n];

    printf("Matriks A\n");
    for (i = 0; i < n; i++) {
        for (j = 0; j < n; j++) {
            scanf("%d", &A[i][j]);
        }
    }

    printf("Matriks B\n");
    for (i = 0; i < n; i++) {
        for (j = 0; j < n; j++) {
            scanf("%d", &B[i][j]);
        }
    }
    
    for (i = 0; i < n; i++) {
        for (j = 0; j < n; j++) {
            C[i][j] = 0;
        }
    }
    for (i = 0; i < n; i++) {
        for (j = 0; j < n; j++) {
            for (k = 0; k < n; k++) {
                C[i][j] += A[i][k] * B[k][j];
            }
        }
    }

    printf("\nMatriks AXB\n");
    for (i = 0; i < n; i++) {
        for (j = 0; j < n; j++) {
            printf("%d ", C[i][j]);
        }
        printf("\n");
    }
 
    return 0;
}

File: MODUL 6/CLASS/PRAK605-2410817110003-FerianDestaAshadi.py
n = int(input(""))

A = []
B = []

print("Matriks A")
for i in range(n):
    A.append(list(map(int, input().split())))
    
print("Matriks B")
for i in range(n):
    B.append(list(map(int, input().split())))
    
C = [[0 for _ in range(n)] for _ in range(n)]

for i in range(n):
    for j in range(n):
        for k in range(n):
            C[i][j] += A[i][k] * B[k][j]
            
print("\nMatriks AXB")
for i in range(n):
    print(" ".join(map(str, C[i])))